wukong/request.py

```diff
@@ -1,10 +1,12 @@
 from wukong.zookeeper import Zookeeper
-from requests.exceptions import ConnectionError
 from wukong.errors import SolrError
 import requests
 import random
 import json
 import time
+import logging
+
+log = logging.getLogger(__name__)
```

Comment:
I think in other files we use `logger = logging.getLogger(__name__)`. Perhaps worth changing it for the sake of consistency?